traffic_portal/test/end_to_end/DeliveryServiceRequests/delivery-service-requests-spec.js

```diff
@@ -144,6 +144,7 @@ describe('Traffic Portal Delivery Service Requests', function() {
 		pageData.deleteButton.click();
 		pageData.confirmWithNameInput.sendKeys(mockVals.xmlId + ' request');
 		pageData.deletePermanentlyButton.click();
+		browser.sleep(250);
 		expect(browser.getCurrentUrl().then(commonFunctions.urlPath)).toEqual(commonFunctions.urlPath(browser.baseUrl)+"#!/delivery-service-requests");
```

Comment:
What's the problem being solved here? A comparison before the promise resolves? Or is `browser.getCurrentUrl()` returning the wrong value if you don't wait a bit first? If it's the former I'm gonna open an issue to get this stuff converted to `async`/`await`, because that'd make waiting for all these things much easier.

Reply:
Based off the test I was working on this morning, it seems to me that `expect()` will wait until promises are resolved before performing its comparisons, so the issue is just `browser.getCurrentUrl()` needs a bit of wait before it returns the correct value. But I couldn't find anything to corroborate that online, so ¯\\\__(ツ)__/¯

Reply:
As far as I could tell from their terrible docs, it doesn't do anything special with Promises. I guess I can always just try it, and if I'm wrong things should randomly break.